tokio-buf/Cargo.toml

```diff
@@ -17,6 +17,10 @@ Asynchronous stream of byte buffers
 categories = ["asynchronous"]
 
 [dependencies]
-bytes = { version = "0.4.10", features = [ "either" ] }
+bytes = { version = "0.4.10" }
 either = "1.5"
```

Comment:
This dep should only be included with the `ext` feature.

Reply:
Don't think this is possible since the core trait requires `bytes::Buf`.